site/en/blog/lighthouse-9-0/index.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: 'layouts/blog-post.njk'
+title: What's new in Lighthouse 9.0
+authors:
+  - brendankenny
+date: 2021-11-12
+description: >
+  Lighthouse 9.0 arrives with a refreshed report and a preview of support for user flows.
+hero: 'image/MtjnObpuceYe3ijODN3a79WrxLU2/hcf9uCrSxA4VNMiXHsVn.png'
+alt: 'A new Lighthouse 9.0 report'
+tags:
+  - new-in-lighthouse
+  - lighthouse
+  - chrome-98
+---
+
+Lighthouse is a website auditing tool that helps developers with opportunities and diagnostics to improve the user experience of their sites.
+
+Lighthouse 9.0 is available immediately on the command line and in [Chrome Canary](https://www.google.com/chrome/canary/).
+It will land in Chrome stable in Chrome 98 and be available in [PageSpeed Insights](https://developers.google.com/speed/pagespeed/insights/) within a week.
```

Comment:
I would rework "within a week" to an absolute timeframe so that people reading the post after this timeframe won't think the process is still in flight after it has landed.